src/ToolbarService.php

```diff
@@ -67,6 +67,7 @@ class ToolbarService
         ],
         'forceEnable' => false,
         'safeTld' => [],
+        'ignorePaths' => null,
```

Comment:
Maybe this is cakephp convention, but `ignorePaths` doesn't feel like a pattern to me.

Up to you, it's not a major api.

Reply:
We could add a `Regex` suffix to the option name but I don't remember similar being done elsewhere.

Reply:
I was thinking `ignorePathsPattern` but whatever. If this is convention then it's convention.